### PR TITLE
Correct broken links in guides

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/guides/identity-lifecycles/configure-user-stores-for-scim2-based-inbound-provisioning.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Now suppose you need to add a secondary userstore, OpenLDAP. In this scenario, c
 
 2.  Navigate to **Main** >  **Identity** > **Claims** > **List**. For more information on configuring claims, see [Configure Claims](../../../guides/dialects/configure-claims). Click on either **urn:ietf:params:scim:schemas:core:2.0:User** or **urn:ietf:params:scim:schemas:core:2.0** claim dialect (depending on which SCIM claim you need to update) and find the mapped local claim of the SCIM claim that you need to edit. 
 
-![claim-list-scim2](../../../assets/img/guides/claim-list-scim2.png)
+    ![claim-list-scim2](../../../assets/img/guides/claim-list-scim2.png)
```
Comment (dewniMW) [inline en/docs/guides/identity-lifecycles/configure-user-stores-for-scim2-based-inbound-provisioning.md:27]:
```suggestion
    ![List of claims available for SCIM dialect](../../../assets/img/guides/claim-list-scim2.png)
```

File: en/docs/guides/identity-lifecycles/rule-based-provisioning.md [hunk 2]
```diff
@@ -41,9 +41,9 @@ Follow the steps given below to configure rule-based provisioning in WSO2 Identi
 
 ## Set up XACML rules
 
-1.  Click on **Policy Administration** under the **Entitlement\>PAP**
-    section on the **Main** tab of the management console.
-2.  Since this sample scenario is based on role, we select the policy
+1.	Click on **Policy Administration** under the **Entitlement\>PAP** section on the **Main** tab of the management console.
```
Comment (dewniMW) [inline en/docs/guides/identity-lifecycles/rule-based-provisioning.md:44]:
```suggestion
1.	Click on **Policy Administration** under the **Entitlement** > **PAP** section on the **Main** tab of the management console.
```

File: en/docs/guides/identity-lifecycles/self-registration-workflow.md [hunk 3]
```diff
@@ -186,6 +185,7 @@ Next, you can try out self-registration.
 
 3.  Fill in the user details, provide consent to share the requested
     information and then click **Register**.
+    
     ![self-signup-form](../../../assets/img/guides/self-signup-form.png)
```
Comment (dewniMW) [inline en/docs/guides/identity-lifecycles/self-registration-workflow.md:189]:
```suggestion
    ![Self sign up form](../../../assets/img/guides/self-signup-form.png) 
```

File: en/docs/guides/login/oidc-token-encryption.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ Make the following changes to the created service provider.
 
 This section guides you through obtaining an encrypted ID token and decrypting it using a simple java program. Alternatively, you can use the WSO2 IS playground sample application to decrypt the token. For instructions, see [Decrypt the ID token](../../../guides/login/oidc-token-decryption).
 
-1. See [OAuth Grant Types](../../access-delegation/oauth-grant-types) and try out one of the grant types with the `openid` scope to obtain an access token.
+1. See [OAuth Grant Types](../../access-delegation/authorization-code) and try out one of the grant types with the `openid` scope to obtain an access token.
```
Comment (dewniMW) [inline en/docs/guides/login/oidc-token-encryption.md:58]: this doc doesn't contain all the grant types. IMO we need to maintain a doc for all grant types. WDYT?

File: en/docs/guides/login/saml-app-config-advanced.md [hunk 5]
```diff
@@ -169,7 +169,7 @@ Enable **Assertion Encryption**, if you wish to encrypt the assertion.
...
 
-WSO2 Identity Server supports both **SAML Back-Channel Logout** and **SAML Front-Channel Logout** methods. By default, when you select **Enable Single Logout**, the Back-Channel Logout is enabled . In order to enable **SAML Front-Channel Logout**, you can either select **Front-Channel Logout (HTTP Redirect Binding)** or **Front-Channel Logout (HTTP POST Binding)** .
+WSO2 Identity Server supports both **SAML Back-Channel Logout** and **SAML Front-Channel Logout** methods. By default, when you select **Enable Single Logout**, the Back-Channel Logout is enabled . In order to enable **SAML Front-Channel Logout**, you can either select **Front-Channel Logout (HTTP Redirect Binding)** or **Front-Channel Logout (HTTP POST Binding)**.
```
Comment (dewniMW) [inline en/docs/guides/login/saml-app-config-advanced.md:172]:
```suggestion
WSO2 Identity Server supports both **SAML Back-Channel Logout** and **SAML Front-Channel Logout** methods. By default, when you select **Enable Single Logout**, the Back-Channel Logout is enabled. In order to enable **SAML Front-Channel Logout**, you can either select **Front-Channel Logout (HTTP Redirect Binding)** or **Front-Channel Logout (HTTP POST Binding)**.
```

File: en/docs/guides/mfa/2fa-email-otp.md [hunk 6]
```diff
@@ -56,8 +56,8 @@ This page guides you through configuring [two-factor authentication](../../../re
     CaptureAndUpdateEmailAddress = true
     showEmailAddressInUI = true
     ```
-    !!! info
-        For information on each of these configurations, see [Email OTP Configurations](../email-otp-config-advanced)
+    <!--!!! info
```
Comment (dewniMW) [inline en/docs/guides/mfa/2fa-email-otp.md:59]: why is this commented?

File: en/docs/guides/login/oidc-token-encryption.md (hunk 4 above)
Comment (gomathyk) [reply en/docs/guides/login/oidc-token-encryption.md:58]: +1 
Will create an overview page for a better redirections. Created https://github.com/wso2/product-is/issues/11990 to track this.

File: en/docs/guides/mfa/2fa-email-otp.md (hunk 6 above)
Comment (gomathyk) [reply en/docs/guides/mfa/2fa-email-otp.md:59]: We don't have a page with details on the advanced configurations.

**Review by dewniMW — APPROVED** [review]

**Pull request merged**